tests/lib/external/snapd-testing-tools/tests/tests.pkgs/task.yaml

```diff
@@ -51,3 +51,11 @@ execute: |
     tests.pkgs noexist test-snapd-pkg-1 2>&1 | MATCH 'tests.pkgs: unknown command noexist'
     tests.pkgs -install test-snapd-pkg-1 2>&1 | MATCH 'tests.pkgs: unknown option -install'
 
+    # Install the test pkg and check it is installed and query it
```

Comment:
Install the test package`,` verify `its installation` and query it.
Run this only on amd64 and skip Trusty.

Reply:
done